### ISSUE TITLE
reproduce SLOWFAST_EPIC_SOUNDS.pyth results

### ISSUE THREAD

**Issue by Acie7J** [epic-kitchens/epic-sounds-annotations#12]
Hi, I wanna reproduce your SLOWFAST_EPIC_SOUNDS.pyth results and I run like this:

```
python tools/run_net.py --cfg configs/EPIC-Sounds/slowfast/SLOWFASTAUDIO_8x8_R50.yaml NUM_GPUS 4 OUTPUT_DIR ./output EPICSOUNDS.AUDIO_DATA_FILE ../../data/EPIC_audio.hdf5 EPICSOUNDS.ANNOTATIONS_DIR ../ TRAIN.CHECKPOINT_FILE_PATH ./SLOWFAST_VGG.pyth
```

But only got 52.30 on local eval set and 52.31 on the test set, while your  SLOWFAST_EPIC_SOUNDS.pyth can achieve 53.73 on local eval set and 54.80 on the test set.
Did I miss something?

**Comment by JacobChalk** [comment]
The training script looks correct, so this may be due to test time augmentations. Have you tried running:

```
python tools/run_net.py \
--cfg configs/EPIC-Sounds/slowfast/SLOWFASTAUDIO_8x8_R50.yaml \
TRAIN.ENABLE False \
TEST.ENABLE True \
NUM_GPUS num_gpus \
OUTPUT_DIR /path/to/outpur_dir \
EPICSOUNDS.AUDIO_DATA_FILE /path/to/EPIC_audio.hdf5 \
EPICSOUNDS.ANNOTATIONS_DIR /path/to/annotations # This is Val or Test \
TEST.CHECKPOINT_FILE_PATH /path/to/experiment_dir/checkpoints/checkpoint_best.pyth
```

This will run the test script, which performs test time augmentation and divides each annotation into 5 sub-clips and then averages the predictions. This should hopefully provide the additional performance you're looking for!
